doc/release-notes.md

```diff
@@ -50,33 +53,246 @@ is not recommended.
-- [`3661f36bbd`](https://github.com/dashpay/dash/commit/3661f36bbd) Merge #14416: Fix OSX dmg issue (10.12 to 10.14) (#4177)
-- [`4f4bda0557`](https://github.com/dashpay/dash/commit/4f4bda0557) depends: Undefine `BLSALLOC_SODIUM` in `bls-dash.mk` (#4176)
-- [`575e0a3070`](https://github.com/dashpay/dash/commit/575e0a3070) qt: Add `QFont::Normal` as a supported font weight when no other font weights were found (#4175)
-- [`ce4a73b790`](https://github.com/dashpay/dash/commit/ce4a73b790) rpc: Fix `upgradetohd` help text (#4170)
-- [`2fa8ddf160`](https://github.com/dashpay/dash/commit/2fa8ddf160) Translations 202105 (add missing) (#4169)
+Quorum rotation
+--------------
+InstantSend quorums will now use a new quorum type and a new algorithm for
+establishing quorums, see [DIP-0024](https://github.com/dashpay/dips/blob/master/dip-0024.md).
+
+Quorum rotation is activated via a BIP9 style hard fork that will begin
+signalling on August 15, 2022 using bit 7. New quorums will start forming in
+1152-1440 block range after the activation. Any nodes that do not upgrade by
+that time will diverge from the rest of the network.
+
+Deterministic InstantSend
+-------------------------
+Deterministically verifying InstantSend locks at any point in the future has
+been added to support Dash Platform. This update introduces versioning to
+InstantSend messages and adds quorum information to them. While the previous
+design was sufficient for core chain payments, the platform chain will benefit
+from this enhanced verification capability. Details about deterministic
+InstantSend are provided in [DIP-0022](https://github.com/dashpay/dips/blob/master/dip-0022.md).
+
+Deterministic InstantSend will be activated with the DIP0024 hard fork.
+
+Governance
+----------
+Several improvements have been made to Dash’s DAO governance system.
+The governance proposal fee has been reduced from 5 Dash to 1 Dash following
+a vote by masternode owners to do so. For improved security and flexibility,
+proposal payouts to pay-to-script-hash (P2SH) addresses are now supported.
+
+These changes wil be activated with the DIP0024 hard fork.
+
+Governance proposals can now be viewed in GUI Governance tab (must be enabled
+in Preferences first).
+
+Initial Enhanced Hard Fork support
+----------------------------------
+The masternode hard fork signal special transaction has been added as the first
+step in enabling an improved hard fork mechanism. This enhancement enables
+future hard forks to be activated quickly and safely without any
+“race conditions” if miners and masternodes update at significantly different
+speeds. Effectively there will be a masternode signal on chain in addition to
+the miner one to ensure smooth transitions. Details of the enhanced hard fork
+system are provided in [DIP-0023](https://github.com/dashpay/dips/blob/master/dip-0023.md).
+
+Network improvements
+--------------------
+We implemented and backported implementations of several improvement proposals.
```

Comment:
```suggestion
We implemented and backported implementations of several improvement proposals. These enhancements provide bandwidth savings for header synchronization, expand the type of addresses that can be relayed between peers, and introduce a Bloom filter alternative that uses compression to minimize filter size.

```

Reply:
Can you refer the the PR / DIP for more information in each feature mentioned here?

Reply:
DIP/BIPs are listed right below this part

Reply:
Yeah, I didn't include them because they are listed in the same order as the links below